src/txn/txn.c

```diff
@@ -2809,14 +2819,30 @@ __wt_verbose_dump_txn_one(
         break;
     }
 
+    WT_ERR(__wt_scr_alloc(session, 2048, &snapshot_buf));
+    WT_ERR(__wt_buf_fmt(session, snapshot_buf, "%s", "["));
+    for (i = 0; i < txn->snapshot_data.snapshot_count; i++) {
```

Comment:
Can we add a python test for this functionality? Checking the snapshot array is correctly outputted. We can start a bunch of transactions and then call session.dump().